### DISCUSSION TITLE
Let’s start a Krita Ai diffusion user group!

### DISCUSSION THREAD

**Discussion by minsky91** [Acly/krita-ai-diffusion#850] [General]
Hello everyone! I have been thinking, why don’t we start a dedicated Krita Ai diffusion user group? I believe there are enough users already, and the momentum is good to create such a group. This github page is mostly for support and development questions, and the spectrum of its subscribers/visitors is limited by definition, so I feel this would be a natural step, if we want this wonderful tool to become more popular and adopted more widely. 

The idea is to gather people, users of the plugin and those who are just curious about it, share our know-how, tips and tricks, images we create with Krita and AI diffusion, as well as relevant news and imaging ideas. I am thinking of establishing this group as a reddit sub, although other options like Facebook or discord are also on the table. A sub on reddit as the group’s location has that advantage, in my view, that we could regularly place some of our most important posts on the big r/StableDiffusion sub, where the plugin has already gotten a good reputation among their members. We will most likely start small, but with time hopefully gather more and more people.

What do you think, folks? Upvote this message if you support this idea, make your own suggestions!
If the idea gets enough upvotes, I will create a poll with more specific questions. And of course, I will post a similar message on the SD reddit sub as well, to ask people’s opinions there.

**Comment by parella007** [comment]
Great idea!

**Comment by lovemakerz** [comment]
1000% Discord !  ^^

**Comment by Acly** [comment]
There's a Discord server now! Invite link: https://discord.gg/pWyzHfHHhU

**Comment by ibow1983** [comment]
good idea！！
